tools/profiles/mprintf.json

```diff
@@ -0,0 +1,8 @@
+{
+    "GCC_ARM": {
+       "common": ["-DMBED_MINIMAL_PRINTF"],
+        "ld": ["-Wl,--wrap,printf", "-Wl,--wrap,sprintf", "-Wl,--wrap,snprintf",
```

Comment:
I'm assuming no other *printf functions are being used in mbed + pdmc use case that result in pulling something from STDLIB.

Reply:
No other *printf function is used.